src/VodafoneChannel.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace NotificationChannels\Vodafone;
+
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification;
+
+class VodafoneChannel
+{
+    /**
+     * @var string Vodafone's API endpoint
+     */
+    protected $endpoint = 'https://www.smsertech.com/apisend';
+
+    /**
+     * Send the given notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     *
+     * @throws \NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        $message = $notification->toVodafone($notifiable, $notification);
+
+        $fields = [
```

Comment:
Can we please use guzzle or another modern HTTP client here.

for example with guzzle the block below is just:

```
try {
	(new Client())->post(['form_params' => [/* unencoded data here */]);
} catch($e) {
	// handle error
}

Reply:
I'd also prefer it if this was split out into a separate client class and bound in the service container.